js/src/forum/components/PostStream.js

```diff
@@ -296,18 +295,17 @@ export default class PostStream extends Component {
    *
    * @param {Integer} index
    * @param {Boolean} animate
-   * @param {Boolean} bottom Whether or not to scroll to the bottom of the post
-   *     at the given index, instead of the top of it.
+   * @param {Boolean} reply Whether or not to scroll to the reply placeholder.
    * @return {jQuery.Deferred}
    */
-  scrollToIndex(index, animate, bottom) {
-    const $item = this.$(`.PostStream-item[data-index=${index}]`);
-
-    return this.scrollToItem($item, animate, true, bottom).then(() => {
-      if (index == this.stream.count() - 1) {
-        this.flashItem(this.$('.PostStream-item:last-child'));
-      }
-    });
+  scrollToIndex(index, animate, reply) {
+    if (reply) {
+      const $placeholder = this.$('.PostStream-item:last-child');
```

Comment:
Couldn't we call `goToIndex()` with `this.count()` in `goToNumber()`, that way then this bit of code gets executed, `index` already points to the placeholder? We can then remove the `if/else` block and just set the `reply` parameter to `scrollToItem`

Reply:
The code is nicer, but unfortunately it's a bit glitchy when an event post has been created, but we haven't scrolled to it yet. When ~battling~ working with PostStream over the past few months, I've found that being as explicit as possible is best.

That being said, I rewrote that bit to be a little cleaner.

Reply:
Good enough then :+1: